include/nbl/ext/DebugDraw/CDraw3DLine.h

```diff
@@ -6,6 +6,7 @@
 #define _NBL_EXT_C_DRAW_3D_LINE_INCLUDED_
 
 #include "nabla.h"
+#include "nbl/ext/DebugDraw/Draw3DLineShaders.h"
```

Comment:
Your separation of recording into cmd buf and drawing is good direction but it needs even more usage flexibility.
First of all it should not care how renderpass looks like. We should get rid of any drawing (actual submits) from this extension. Leave just recording to cmd buf. It should not care about swapchains, queues, framebuffers or renderpasses.
API should be more-less:
* addLine()
* setData()
* addBox() (originally it had `queueBox` method)
* recordIntoCommandBuffer(cmdbuf, graphics_pipeline)
While recording, have cmd buf provided from outside and assume beginRenderpass() was already recorded (write docs). `assert()` that provided cmdbuf is in recording state.
Dont create your own graphics pipeline. Create just IGPURenderpassIndependentPipeline. User should create graphics pipeline from it and pass into recordIntoCommandBuffer()